services/backup/backup_service.go

```diff
@@ -278,8 +292,8 @@ func (s *Service) RestoreBackup(ctx context.Context, serviceID, artifactID strin
 
 		if params.ServiceType == models.MySQLServiceType && params.DBVersion != "" {
 			if params.DBVersion != dbVersion {
-				return errors.Errorf("incompatible service: artifact db version %q != db version %q",
-					params.DBVersion, dbVersion)
+				return fmt.Errorf("%w: incompatible service: artifact db version %q != db version %q",
```

Comment:
maybe it's better to use `errors.Wrap` ?
Error message here will be like `incompatible service: incompatible service: artifact db version %q != db version %q`, so it's better to not  duplicate that we already have in wrapped error.

Reply:
Here and in other places

Reply:
done